src/facilities.js

```diff
@@ -41,13 +44,84 @@ class Facilities {
     this._sdk = sdk;
   }
 
+  /**
+   * Creates a new facility
+   *
+   * API Endpoint: '/facilities'
+   * Method: POST
+   *
+   * @param {Object} options
+   * @param {string} [options.address1]
+   * @param {string} [options.address2]
+   * @param {string} [options.city]
+   * @param {string} [options.geometryId] UUID corresponding with a geometry
+   * @param {string} options.name
+   * @param {string} options.organizationId UUID corresponding with an organization
+   * @param {string} [options.state]
+   * @param {string} options.timezone
+   * @param {number} [options.weatherLocationId]
+   * @param {string} [options.zip]
+   *
+   * @returns {Promise}
+   * @fulfill {Facility} Information about the new facility
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.facilities
+   *   .create({
+   *     address: '221 B Baker St, London, England',
+   *     name: 'Sherlock Homes Museum',
+   *     organizationId: 25
+   *   })
+   *   .then((facilities) => console.log(facilities));
+   *   .catch((err) => console.log(err));
+   */
+  create(facility = {}) {
+    const requiredFields = ['organizationId', 'name', 'timezone'];
+
+    for (let i = 0; requiredFields.length > i; i++) {
```

Comment:
Is there any reason you're iterating over the array in that manner instead of `['organizationId', 'name', 'timezone'].forEach()`? Just curious if there was a compatibility concern. I do think the `for()` looks cleaner, but just curious!

Reply:
Using a for loop in this case because when you return, it returns `create` instead of the anonymous function that you'd be using in forEach. You could make it work with forEach or map, but it would be a bit more code